content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
+## Scope
 
-```hcl
-{
-    us-east-1 = "image-1234"
-    us-west-2 = "image-4567"
-}
-```
+You can [set workspace-specific variables](/docs/cloud/workspaces/managing-variables.html#workspace-specific-variables) inside each workspace or [create variable sets](/docs/cloud/workspaces/managing-variables.html#variable-sets) to reuse the same variables across multiple workspaces. For example, you could define a variable set of provider credentials and automatically apply it to several workspaces, rather than manually defining credential variables in each. Changes to variable sets instantly apply to all appropriate workspaces, saving time and reducing errors from manual updates.
 
-HCL can be used for Terraform variables, but not for environment variables. The HCL code you enter for values is interpreted by the same Terraform version that performs runs in the workspace. (See [How Terraform Cloud Uses Variables](#how-terraform-cloud-uses-variables) below.)
+You can create a global variable set by applying it to all current and future workspaces. Terraform Cloud does not allow global variable sets to contain conflicting variables with same type and the same key. When non-global variable sets contain conflicting variables and you apply them to the same workspace, the variable set applied most recently takes precedence. Refer to the [precedence documentation](#precedence) for more details.
 
-### Sensitive Values
 
-Terraform often needs cloud provider credentials and other sensitive information that shouldn't be widely available within your organization.
+## Precedence
 
-To protect these secrets, you can mark any Terraform or environment variable as sensitive data by clicking its "Sensitive" checkbox (visible when editing).
+> **Hands On:** The [Manage Multiple Variable Sets in Terraform Cloud](link) tutorial on HashiCorp Learn shows how to manage multiple variable sets and demonstrates variable precedence.
 
-Marking a variable as sensitive prevents anybody (including you) from viewing its value in the variables section of the workspace in Terraform Cloud's UI or with its Variables API endpoint.
+If variables from `.auto.tfvars` files have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
 
-Users with permission to read and write variables can set new values for sensitive variables.  No other attribute of a sensitive variable can be modified. To update other attributes, delete the variable and create a new variable to replace it.
+There are two additional cases when a workspace may have conflicting variables with the same key. You may use workspace-specific variables to [overwrite variables from variable sets](/docs/cloud/workspaces/managing-variables.html#overwrite-variable-sets) or there may be two variables from different variable sets with the same key.
```

Comment:
this section feels unclear to me. maybe we can organize the different overwrite scenarios under subheadings? something like
- conflicts between workspace-specific variables and variable sets
- conflicts between multiple variable sets

Reply:
I think usually when I look for a precedence section I expect a list of all the ways you can set the thing from highest to lowest precedence.

Reply:
okay I'm pushing up a big change here - you're right. This section is not working. I'm hoping what I'm pushing up is better!